nautobot/docs/user-guide/administration/installation/nautobot.md

```diff
@@ -266,6 +266,45 @@ Nautobot does not come with any predefined user accounts. You'll need to create
 nautobot-server createsuperuser
 ```
 
+## Build Nautobot 2.0 UI
+
+Nautobot 2.0 introduces a new user interface. This interface is built using [React](https://react.dev/) and requires Node.js version 18 or higher to build the UI. 
+
+### Install Node.js
+
+Node.js version 18 is only listed on package manager repositories for Debian 12 Bookworm, Ubuntu 22.10 kinetic and RHEL 8.
+
+Debian Bookworm or Ubuntu kinetic or higher:
+
+```no-highlight
+sudo apt-get -y install npm
+```
+
+RHEL 8/9:
+
+```no-highlight
+sudo dnf module install nodejs:18/common
+```
+
+Older versions of Debian and Ubuntu must install Node.js from source or the [NodeSource](https://github.com/nodesource/distributions#deb) repositories:
+
+```no-highlight
+curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash -
+sudo apt-get install -y npm
```

Comment:
Is there a good reason we're using this approach in our Docker build currently? Is python-slim still on an older version of debian/ubuntu that can't use the streamlined approach above?

Reply:
Bookworm was only released a month ago but it looks like the python `VERSION-slim` images switched their base to bookworm last month too. We can probably switch to using the official repos in our dockerfile now.

Reply:
#4110